webapp/src/components/MeetingQuickView.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Alert,
+  Link as MuiLink,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import { LoadingButton } from '@mui/lab';
+import React, { FormEventHandler, useContext, useState } from 'react';
+
+import { EntityId, Meeting as APIMeeting } from '../types/api';
+import { formatDate, formatTime } from '../helpers/dateTime';
+import SessionContext from './SessionContext';
+import useApiData from '../helpers/useApiData';
+
+interface MeetingQuickViewProps {
+  meeting: APIMeeting;
+}
+
+const MeetingQuickView = ({ meeting }: MeetingQuickViewProps) => {
+  const { principalId } = useContext(SessionContext);
+  const [changedMeetingNoteIds, setChangedMeetingNoteIds] = useState<
+    EntityId[]
+  >([]);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isSavingMeetingNote, setIsSavingMeetingNote] = useState(false);
+  const [isSuccessIndicated, setIsSuccessIndicated] = useState(false);
```

Comment:
What was successful? This variable name forces me to have to look at its usage to understand what it means.

Reply:
Adding a new agenda item.  I'll have to change the variable name since this refers to behavior of the create meeting note form.